config/deployment/dev-unstable.json

```diff
@@ -0,0 +1,22 @@
+{
+  "services": {
+    "ceramic": {
+      "mode": "client",
+      "apiURL": "http://localhost:7007"
+    },
+    "ceramic2": {
+      "mode": "client",
+      "apiURL": "https://ceramic-dev.3boxlabs.com",
+      "anchorServiceAPI": "https://cas-dev.3boxlabs.com",
```

Comment:
At some point we should add a cas just for testing so we can control when anchoring happens or other config.